data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+		has "main plot completed"
+		has "language: Wanderer"
+	on offer
+		conversation
+			branch "came alone"
+				"ships: Transport" * 6 + "ships: Light Freighter" * 3 + "ships: Heavy Freighter" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 <= 18
+			`You find Si-Ra and Hi-Ye waiting for you on the spaceport. Si-Ra starts speaking to you, "You have brought too many ships, come back with only your flagship, or a few lighter ships."`
+				defer
+			label "came alone"
+			`You find Si-Ra and Hi-Ye waiting for you on the spaceport. Si-Ra asks, "Are you ready for war, <last>?"`
+			choice
+				`	"Not yet. I will come back when I am."`
+					goto later
+				`	"I was born ready."`
+			`	"And may you die ready as well." Si-Ra comes closer and takes you aside to speak to you, privately. "I have managed to buy myself a ship with the money from our last business opportunity. It is much stronger than the old one! And as a warlord, Hi-Ye has a strong fleet as well, though he will not use it here - only his flagship. You may want to consider this seriously. You will first have to duel me and then him. Don't forget you can call the fight off and come back once you're ready by giving us a symbolic fee."`
+			choice
+				`	"You don't scare me."`
+				`	"On second thought, I'll come back later."`
+					goto later
+				`	"Now that it's here I'm starting to realize I don't want to be a part of your way. I'm out.`
+					decline
+			`	"That is good! Let us fight, on the same terms of honor as before. Take off when you're ready, I will be waiting for you. Once you defeat me you will be able to fight the Tejolis."`
+				accept
+			label later
+			`	"Be sure to come back soon. War cannot wait forever."`
+				defer
+	npc save disable
+		to despawn
+			has "unfettered: defeated Si-Ra"
+		government "Hai (Unfettered Challenge)"
+		ship "Enhanced Shield Beetle (Pulse)" "Intrepid II"
+		personality disables staying heroic target launching
+		conversation
+			`"You have defeated me, But Hi-Ye will not be as easy to beat! Land to repair, and rhen take off when you're be ready to face him."`
+			action
+				set "unfettered: defeated Si-Ra"
+	npc save disable
+		to spawn
+			has "unfettered: defeated Si-Ra"
+		government "Hai (Unfettered Challenge)"
+		ship "Shield Beetle Prototype (Black)" "Tejolis"
+		personality disables staying heroic target launching
+		conversation
+			action
+				"reputation: Hai (Unfettered)" += 100
+				event "shield beetles accessible"
+			`Now that the fight is over, they both hail you, and He-Yi speaks first. "You sure know how to fight! I feel now maybe we are the ones who need training. I feel like it is only fair you give us tips this time."`
+			choice
+				`	"You were overconfident, and that's what let you down."`
+					goto mistake
+				`	"A good fight, but you just had no chance against my superior firepower."`
+				`	"You lacked coordination with each other."`
```

Comment:
This seems like a weird thing to say because the mission is literally designed such that they _cannot_ coordinate.

Reply:
Oh right, it was 1v2 before I forgot to change thos